lib/scss/components/cta-list/index.scss

```
@@ -0,0 +1,7 @@
+@charset 'UTF-8';
+
+@import
+'decanter/core/src/scss/decanter-no-markup',
+'../../config.scss',
+'cta-list.scss';

```

```suggestion
'../../config',
'cta-list';
```
extensions are not allowed.